common/components/common/groups/AboutGroupDisplay.jsx

```diff
@@ -198,79 +95,35 @@ class AboutGroupDisplay extends React.PureComponent<Props, State> {
-                positions={this.state.group && this.state.group.group_positions}
-                positionToJoin={this.state.positionToJoin}
-                showModal={this.state.showJoinModal}
-                handleClose={this.confirmJoinGroup.bind(this)}
-              />
-
               {!this.props.viewOnly && this._renderContactAndVolunteerButtons()}
 
             </div>
 
             <div className="AboutProjects_tabs">
 
-              <a onClick={() => this.changeHighlighted('details')} className={this.state.tabs.details ? 'AboutProjects_aHighlighted' : 'none'}href="#group-details">Details</a>
-
-              {group && !_.isEmpty(group.group_positions) &&
-              <a onClick={() => this.changeHighlighted('skills')} className={this.state.tabs.skills ? 'AboutProjects_aHighlighted' : 'none'} href="#positions-available">Skills Needed</a>
-              }
-
             </div>
           </div>
 
           <div className='AboutProjects-details'>
             <div id='group-details'>
               {group.group_description}
-              {!_.isEmpty(group.group_description_solution) && 
-                <React.Fragment>
-                  <div>
-                    <br></br>
-                    {group.group_description_solution}
-                  </div>
-                </React.Fragment>
-              }
-              {!_.isEmpty(group.group_description_actions) && 
-                <React.Fragment>
-                  <div>
-                    <br></br>
-                    {group.group_description_actions}
-                  </div>
-                </React.Fragment>
-              }
             </div>
 
             <div className='AboutProjects-skills-container'>
-
-              {group && !_.isEmpty(group.group_positions) &&
-                <div className='AboutProjects-skills'>
-                  <p id='skills-needed' className='AboutProjects-skills-title'>Skills Needed</p>
-                  {group && group.group_positions && group.group_positions.map(position => <p>{position.roleTag.display_name}</p>)}
-                </div>
-              }
-
-              {group && !_.isEmpty(group.group_technologies) &&
-                <div className='AboutProjects-technologies'>
-                  <p className='AboutProjects-tech-title'>Technologies Used</p>
-                  {group && group.group_technologies && group.group_technologies.map(tech => <p>{tech.display_name}</p>)}
-                </div>
+              {!_.isEmpty(this.state.issueAreas) &&
+              <div className='AboutProjects-skills'>
+                <p id='skills-needed' className='AboutProjects-skills-title'>Issue Areas</p>
+                {this.state.issueAreas && this.state.issueAreas.map((issue: TagDefinition) => <p>{issue.display_name}</p>)}
+              </div>
               }
-
             </div>
+            
+            {this.state.group.group_projects && this._renderProjectList()}
```

Comment:
```suggestion
            <div className="container-fluid">
              <div className="row">
                {this.state.group.group_projects && this._renderProjectList()}
              </div>
            </div>
```
Wrapping all the cards in Bootstrap's container-fluid and row like this should be the fastest way to get cards to display two wide at desktop breakpoints like we want them to. Longer term we'll probably want to standardize the mechanisms of how cards are laid out on their various pages (homepage, Find pages, now About Group...) because we're not consistent (probably my fault, honestly) but this is enough for now.

Reply:
fixed in 419241f